fcl_product_components/components/gne_reference_pane/sass/gne-reference-pane.scss

```diff
@@ -0,0 +1,43 @@
+$include-gne-reference-styles: true !default;
+
+// Default component class.
+$gne-reference-pane-selector: '.pane-fcl-procat-product-gne-reference-pane';
+$gne-reference-selector: '.gne-reference-pane' !default;
+$gne-reference-pane-gne-reference-display: inline-block !default;
+$gne-reference-pane-gne-wrapper-display: $gne-reference-pane-gne-reference-display !default;
+$gne-reference-pane-gne-wrapper-margin: 0 0 0 15px !default;
+$gne-reference-pane-padding: 0.875em;
```

Comment:
Use `.875em` instead. 
Run this through:
`scss-lint filepath` in your terminal from `~/docroot/web`